src/main/java/sirius/pasta/noodle/Invocation.java

```diff
@@ -266,6 +264,13 @@ private void handlePopField() {
         }
     }
 
+    private void handleJumpFalse(int index) {
+        Object value = pop();
+        if (value == null || Boolean.FALSE.equals(value)) {
```

Comment:
Oder `if (!Boolean.TRUE.equals(value))`?

Reply:
I'd keep the original if as suggested